src/Utils/actionCondition.ts

```diff
@@ -140,3 +140,35 @@ export const getUniqueConditions = (actions: CLM.ActionBase[]): CLM.Condition[]
 
     return uniqueConditions
 }
+
+/**
+ * Returns actions using the existing condition with modifications of replaces it with the new condition.
```

Comment:
"*and* replaces it" ?

Reply:
Hmm, yea, wording isn't right. I don't think `and` is better though, it makes it sound like it does A _and_ B, but it just returns actions. The comment was trying to describe the kind of modification the updated actions have. I think the issue was with the word "replaces" should be "replacing". 

Maybe this?
> Returns actions using the existing condition with modifications of existing condition replaced by new condition.

Reply:
> Returns a copy of the input actions, which have `existingCondition` replaced with `newCondition` 

?

Reply:
Nah, it doesn't return copy of the input actions

Reply:
Ah, that's actually a good argument for changing the method name to something like `updateActionConditions`.  It's not obvious from the name that it mutates the input args.

> Updates the input actions by replacing `existingCondition` with `newCondition` ?

Reply:
> it mutates the input args

It doesn't and we should avoid that.

Reply:
Ok, I'm super confused.  I thought you just said it **doesn't** return a copy ?

Reply:
Correct, it only returns array of actions which had the existing condition not copy of entire input.